code/modules/clothing/suits/wintercoats.dm

```diff
@@ -227,8 +235,17 @@
 	desc = "A white winter coat with green markings. Warm, but wont fight off the common cold or any other disease. Might make people stand far away from you in the hallway. The zipper tab looks like an oversized bacteriophage."
 	icon_state = "coatviro"
 	inhand_icon_state = "coatviro"
+/obj/item/clothing/suit/hooded/wintercoat/medical/viro/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/bio,
+
+	)
+
 	hoodtype = /obj/item/clothing/head/hooded/winterhood/medical/viro
 
+
+
```

Comment:
why

Reply:
Fixed, but will post it later as I have no Ethernet currently and writing from phone